engine/src/main/java/com/datatorrent/stram/StramClient.java

```diff
@@ -448,6 +450,14 @@ public void startApplication() throws YarnException, IOException
       }
       String libJarsCsv = copyFromLocal(fs, appPath, localJarFiles.toArray(new String[]{}));
 
+      if (UserGroupInformation.isSecurityEnabled() && customKeystoreConfig != null && customKeystorePath != null) {
+        String[] customSSLConfArr = {customKeystoreConfig, customKeystorePath};
+        String customSSLFileConf = copyFromLocal(fs, appPath, customSSLConfArr);
+        String configFileName = (new java.io.File(customKeystoreConfig)).getName();
+        dag.getAttributes().put(Context.DAGContext.STRAM_HTTP_CUSTOM_CONFIG, configFileName);
```

Comment:
@PramodSSImmaneni "consistency" to me is not a reason to continue with spreading the wrong things. I also want to see that API changes are called out and reviewed in a different way going forward and signed of by more than one committer qualified to do such review.

MASTER_MEMORY_MB:  There are master and worker containers, the user has to understand the difference when specifying memory, hence this is a case where it needs to be qualified within the scope. Attributes are scoped, DAGContext is for platform settings on application level. There is no other HTTP_ setting that would justify additional qualification and it is not ambiguous. While "MANAGEMENT" would be more appropriate than "STRAM", it does not seem necessary in this case.